### PR TITLE
[KCC] Implement a KCC helm solution for subnet 

### PR DESCRIPTION
**Pull request by MingyanZhao**
Implement a KCC helm solution for subnet

### CODE REVIEW

File: config-connector/solutions/iam/helm/subnet/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+name: subnet
+version: 0.1.0
+description: Create a subnet and grant a specific member a role to a member for the subnet
```
Comment (maqiuyujoyce) [inline config-connector/solutions/iam/helm/subnet/Chart.yaml:18]: Nit: Create a subnet and grant `a specific member a role` for the subnet

File: config-connector/solutions/iam/helm/subnet/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+  subnet
+
+## SYNOPSIS
+
+  Config Connector compatible YAML files to create a subnet in your desired project, and grant a specific member a role (default to `roles/compute.networkUser`) for accessing the subnet that just created.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository
```
Comment (maqiuyujoyce) [inline config-connector/solutions/iam/helm/subnet/README.md:15]: Nit: 1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository`:`

File: config-connector/solutions/iam/helm/subnet/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    helm template . --set iamPolicyMember.iamMember=user:name@example.com
+
+    # do a dryrun on your chart and address issues if there are any
+    helm install . --dry-run --set iamPolicyMember.iamMember=user:name@example.com --generate-name
+
+    # install your chart
+    helm install . --set iamPolicyMember.iamMember=user:name@example.com --generate-name
+    ```
+
+1. _Optionaly_, you can costomize optional values by explictly set them when installing the solution:
```
Comment (maqiuyujoyce) [inline config-connector/solutions/iam/helm/subnet/README.md:54]: Nit: costomize -> customize

Comment (maqiuyujoyce) [inline config-connector/solutions/iam/helm/subnet/README.md:54]: Nit: ... by explictly `setting` them ...

File: config-connector/solutions/iam/helm/subnet/Chart.yaml (hunk 1 above)
Comment (MingyanZhao) [reply config-connector/solutions/iam/helm/subnet/Chart.yaml]: Done.

Comment (MingyanZhao) [reply config-connector/solutions/iam/helm/subnet/Chart.yaml]: Done.

File: config-connector/solutions/iam/helm/subnet/README.md (hunk 2 above)
Comment (MingyanZhao) [reply config-connector/solutions/iam/helm/subnet/README.md]: Done.

File: config-connector/solutions/iam/helm/subnet/README.md (hunk 3 above)
Comment (MingyanZhao) [reply config-connector/solutions/iam/helm/subnet/README.md]: Done.

Comment (MingyanZhao) [reply config-connector/solutions/iam/helm/subnet/README.md]: Done.

**Pull request merged**